Lib/cu2qu/ufo.py

```diff
@@ -225,11 +225,15 @@ def fonts_to_quadratic(
     assert len(max_errors) == len(fonts)
 
     modified = False
-    for glyphs in zip(*fonts):
-        name = glyphs[0].name
-        assert all(g.name == name for g in glyphs), 'Incompatible fonts'
+    for name in set().union(*(f.keys() for f in fonts)):
+        glyphs = []
+        my_errors = []
```

Comment:
Nit: let's try to avoid variable names prefixed by "my". In this case I think even `cur_max_errors` is better.